spec/models/utils/load_testing/sms_submission_load_test_spec.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe Utils::LoadTesting::SmsSubmissionLoadTest do
+  let(:setting) { FactoryGirl.build(:setting, incoming_sms_token: "token") }
```

Comment:
Don't need `FactoryGirl` literal here or in the other spec. Please remove from both.